ublk.cpp

```diff
@@ -476,6 +296,22 @@ static int cmd_dev_help(int argc, char *argv[])
 	return EXIT_FAILURE;
 }
 
+static int cmd_dev_recover(int argc, char *argv[])
```

Comment:
We can avoid the user-visible 'recover' command line change, the target type can be retrieved from the pid file from device number, and pid file is required for recover feature.